src/new.rs

```diff
@@ -89,13 +98,24 @@ pub fn create_new_project<P: AsRef<Path>>(dest: P) -> Result<()> {
     Ok(())
 }
 
-pub fn create_new_document(doc_type: &str, name: &str, config: &Config) -> Result<()> {
+pub fn create_post(name: &str, config: &Config) -> Result<()> {
```

Comment:
This function only uses `config` for `source`.  Unless we expect to use more of config, we should probably have this function take a `path` instead.